scipy/stats/tests/test_continuous_basic.py

```diff
@@ -363,6 +363,41 @@ def test_rvs_broadcast(dist, shape_args):
     check_rvs_broadcast(distfunc, dist, allargs, bshape, shape_only, 'd')
 
 
+def test_gh17775_regression():
+    # Regression test for gh-17775. In scipy 1.9.3 and earlier, these tests would fail.
+    # KS one asymptotic sf ~ e^(-(6nx+1)^2 / 18n)
+    # Given a large 32-bit integer n, 6n will overflow in the c implementation.
+    ks = stats.ksone
+    n = 1000000000 # 10**9
+    # Tests use 1/n < x < 1-1/n. Larger x has a smaller sf.
+    # Expected values computed using mpmath to 50 dps and output at 20.
```

Comment:
Could you add the mpmath code in comment? We like to do that. You can look for other examples in stats.